config/develop/glue-job-S3ToJsonS3.yaml

```diff
@@ -0,0 +1,16 @@
+template_path: glue-spark-job.j2
+stack_name: glue-job-S3ToJsonS3
+#{% set BranchOrTagName='main' %}
+parameters:
+  JobRole: !stack_output_external glue-job-role::RoleArn
+  S3BucketName: !stack_output_external bridge-downstream-dev-intermediate-bucket::BucketName
+  BookmarkOption: job-bookmark-disable
+  JobDescription: Convert data to JSONS3 data
+  MaxConcurrentRuns: '150'
+  S3ScriptLocation: s3://{{ stack_group_config.artifact_bucket_name }}/BridgeDownstream/{{ BranchOrTagName }}/glue/jobs/s3_to_json_s3.py
```

Comment:
Unsure, but can BranchOrTagName be used here?

Reply:
No, not for this template. We're discussing a refactor for how this is handled to simplify it. Let's talk about it at the standup tomorrow.